Podfile

```diff
@@ -50,7 +50,7 @@ def wordpress_ui
 end
 
 def gravatar
-  pod 'Gravatar', path: '../Gravatar-SDK-iOS', :testspecs => ['Tests']
+  pod 'Gravatar', path: '../Gravatar-SDK-iOS', :testspecs => ['GravatarTests']
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/58cb1f881b056e3789ec0b60c911c9c04862b811/Podfile#L53"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.</td>
    </tr>
  </tbody>
</table>